_includes/landing-navbar.html

```diff
@@ -21,8 +21,8 @@
             class="navbar-brand navbar-brand-beta"
             style="padding-left: 0"
             href="{{ landing_page_base_url }}"
-            ><strong>Gruntwork</strong>&nbsp;Labs</a
-          >
+            ><strong>Gruntwork</strong>&nbsp&nbsp;Labs
```

Comment:
NIT: Is this change still needed in this PR?

Reply:
Oh, that must be putting it on one line. I don't think it's got a place in this PR! Thanks for calling it out!